aptos-move/framework/aptos-token-objects/sources/collection.move

```diff
@@ -151,6 +151,9 @@ module aptos_token_objects::collection {
             royalty::init(&constructor_ref, option::extract(&mut royalty))
         };
 
+        let transfer_ref = object::generate_transfer_ref(&constructor_ref);
```

Comment:
Awesome! Can you add a comment here explaining that transferring collection objects is disabled as it's meaningless - the new owner wouldn't gain any control over the collection.